src/coreclr/utilcode/debug.cpp

```diff
@@ -387,7 +387,7 @@ bool _DbgBreakCheck(
         OutputDebugStringA("\n");
         OutputDebugStringA(szFile);
```

Comment:
Nit: Should we switch filename to UTF8 as well for consistency?

Reply:
I'll switch this whole block to UTF8 to be consistent.